bootstrap.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+###
+# create swap file
+###
+sudo dd if=/dev/zero of=/swapfile bs=1024 count=1024k
+sudo mkswap /swapfile
+sudo swapon /swapfile
+echo '/swapfile none swap sw 0 0' | sudo tee --append /etc/fstab
+sudo chown root:root /swapfile
+sudo chmod 0600 /swapfile
+
+
+###
+# for Chinese network
+###
+  # rvmsudo gem sources --remove https://rubygems.org/
+  # rvmsudo gem sources -a https://ruby.taobao.org/
+  # sed -i 's!cache.ruby-lang.org/pub/ruby!ruby.taobao.org/mirrors/ruby!' $rvm_path/config/db
+
+
+###
+# bundle
+###
+sudo apt-get install libpq-dev
+sudo apt-get install postgresql postgresql-contrib
```

Comment:
`vagrant ssh` into VM
Try these two apt command to fix installing of pg gem.
Then ` cd /vagrant && bundle`